chain/factory/toolsfactory/input.cue

```diff
@@ -1,4 +1,4 @@
-package tools
+package toolsfactory
```

Comment:
toolsfactory is not a good name..
We should just merge this with scaffoldfactory, and rename it to like:

```

#ScaffoldTools: {
	...
}
```

Reply:
I have rename toolsfactory to ingressfactory, And merge it to cdfactory.
now ingressfactory just for backup. In the future, if we need to automatically install a different version of ingress(such as infra stack), it will continue to be used.